docs/int/quickstart/advanced/quickstart-sdk.md

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 1
+description: Create a DV cluster using the Obol Typescript SDK
+---
+
+# Create a DV using the SDK
+
+:::caution
+
+The Obol-SDK is in an alpha state and should be used with caution.
+:::
+
+This is a walkthrough of using the [Obol-SDK](https://www.npmjs.com/package/@obolnetwork/obol-sdk) to propose a four-node distributed validator cluster for creation using the [DV Launchpad](../../../dvl/intro.md). 
+
+## Pre-requisites
+
+- You have [node.js](https://nodejs.org/en) installed.
+
+## Install the package
+
+Install the Obol-SDK package into your development environment
+
+```bash
+# NPM
+npm install --save @obolnetwork/obol-sdk
+
+# Yarn
+yarn add @obolnetwork/obol-sdk
+```
+
+## Instantiate the client
+
+The first thing you need to do is create a instance of the Obol SDK client. The client takes two constructor parameters:
+
+- The `chainID` for the chain you intend to use.
+- An ethers.js [signer](https://docs.ethers.org/v6/api/providers/#Signer-signTypedData) object.
+
+
+```ts
+import { Client } from "@obolnetwork/obol-sdk";
+import { ethers } from "ethers";
+
```

Comment:
Should we leave it to the SDK User to decide how they want to use this inside a `.js` or a `.ts` file? For example, if i want to write a `.js` file i would either declare `type:module` or use `require` statements. On the other hand if i would like to write a `.ts` file i would need to transpile and run.

Reply:
Oh yeah, my bad, forgot that is not a `js` way to do this, will need to fix but am travelling today. We could also make a tab such that we have js and ts (except I seemed to be having trouble rendering one with docusaurus  at such a nested page level, idk, it's why i didn't put the `npm` and `yarn` commands into separate tabs)